source/include/kickelens.f90

```diff
@@ -35,8 +35,9 @@
   else ! r1 < r2 <= rrelens
     frrelens = one
   endif
-  frrelens = elens_r2(ielens(ix))/rrelens * frrelens
-  yv1(j)=yv1(j)-elens_theta_r2(ielens(ix))*frrelens*xelens/rrelens * oidpsv(j)
-  yv2(j)=yv2(j)-elens_theta_r2(ielens(ix))*frrelens*yelens/rrelens * oidpsv(j)
+  ! 'radial kick'
+  frrelens = elens_theta_r2(ielens(ix))*elens_r2(ielens(ix))/rrelens * frrelens * oidpsv(j)*mtc(j)
+  yv1(j)=yv1(j)-frrelens*xelens/rrelens
+  yv2(j)=yv2(j)-frrelens*yelens/rrelens
```

Comment:
These lines need brackets for consistent rounding.

Reply:
done - I will anyway review the entire code of elenses to make sure that there are no other left-overs of this type